Tasks/ANT/anttask.ts

```diff
@@ -78,14 +80,28 @@ function processAntOutputLine(line) {
 
 async function doWork() {
 
-    function enableCodeCoverage() {
+    function execEnableCodeCoverage(): Q.Promise<string> {
```

Comment:
The code coverage functions should be a separate module instead of a nested function so the main task code is clean.  I realize it was already this way but it's a simple change.